arangod/GeneralServer/H2CommTask.cpp

```diff
@@ -580,9 +581,24 @@ void H2CommTask<T>::sendResponse(std::unique_ptr<GeneralResponse> res,
 
   // this uses a fixed capacity queue, push might fail (unlikely, we limit max streams)
   unsigned retries = 512;
-  while (ADB_UNLIKELY(!_responses.push(tmp))) {
-    std::this_thread::yield();
-    if (--retries == 0) {
+  try {
+    while (ADB_UNLIKELY(!_responses.push(tmp))) {
+      std::this_thread::yield();
+      if (--retries == 0) {
+        LOG_TOPIC("924dc", WARN, Logger::REQUESTS)
```

Comment:
this will fail the Jenkins test as it has the same logID as the log message below.